### PR TITLE
Various docs improvements

### PR DESCRIPTION
**Pull request by sclu1034**

### CODE REVIEW

**Review by Aire-One — DISMISSED** [review]

**Comment by Elv13** [comment]
@psychon Sorry to bother you, but I am still trying to root out of flacky tests. Any idea how to make these lines more reliable? Should we have some way to removing/resetting the shape from a client?

xwindow.c
image

client.c
image

**Review by Elv13 — APPROVED** [review]

File: docs/ldoc.css [hunk 1]
```diff
@@ -47,7 +47,11 @@ p.name {
     font-family: monospace;
 }
 
-kbd, p code, ol code {
+kbd,
+p code,
+ol code,
+ul code,
```
[the remark this answers is not in the document]
Comment (sclu1034) [reply docs/ldoc.css:53]: I wish this wasn't necessary. The lack of `<p>` tags for the first line/paragraph in the rendered item description is just wrong (in terms of "proper" HTML structure).
But it seems that this originates rather deep in LDoc's Markdown handling, so not easy to fix.

**Comment by psychon** [comment]
@Elv13 Uhm.... that `if (!extents)` is a race. This path is taken if QueryExtents fails.https://www.x.org/releases/X11R7.7/doc/xextproto/shape.html does not even document error cases for that request, so.... I guess this can only happen if the window was already destroyed? That's impossible to trigger reliably. The window must destroy itself just in the right moment while AwesomeWM is working on it. Too early and an earlier check would cause the whole window to be skipped. Too late and this request does not fail.

Perhaps it'd be easier to figure out which existing code hits this race and to make this somehow non-racy. But... yeah, dunno how.

The code in `client.c`... well... `xwindow_get_shape` returns `NULL` in two cases where the X11 server has no SHAPE support. That can't be the problem here. The last one is a window which does not have a shape. Shouldn't that be easy to trigger? Just query the shape of a window that did not set a shape on itself? Are there even tests that query the shape of a window that has a shape applied?

File: docs/ldoc.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Elv13) [reply docs/ldoc.css:53]: Yeah, I came to the same conclusion...

**Comment by Elv13** [comment]
> Are there even tests that query the shape of a window that has a shape applied?

Thanks for looking. I don't think we have that yet. I will take care of it soon.

**Review by Aire-One — APPROVED** [review]

**Pull request merged**